tools/depends/native/TexturePacker/src/XBTFWriter.cpp

```diff
@@ -84,18 +77,7 @@ void CXBTFWriter::Cleanup()
 
 bool CXBTFWriter::AppendContent(unsigned char const* data, size_t length)
 {
-  unsigned char *new_data = (unsigned char *)realloc(m_data, m_size + length);
-
-  if (new_data == nullptr)
-  { // OOM - cleanup and fail
-    Cleanup();
-    return false;
-  }
-
-  m_data = new_data;
-
-  memcpy(m_data + m_size, data, length);
-  m_size += length;
+  std::copy(data, data + length, std::back_inserter(m_data));
```

Comment:
```suggestion
  m_data.insert(m_data.end(), data, data + length);
```
This should be a little bit more efficient I assume.

Reply:
Thanks!